.travis.source.yml

```diff
@@ -0,0 +1,122 @@
+language: ruby
+sudo: false
+
+cache:
+  bundler: true
+  directories:
+    - /tmp/cache/unicode_conformance
+    - /tmp/beanstalkd-1.10
+    - node_modules
+    - $HOME/.nvm
+
+services:
+  - memcached
+  - redis-server
+
+addons: &addons
+  postgresql: "9.6"
+
+bundler_args: --without test --jobs 3 --retry 3
+before_install:
+  - "rm ${BUNDLE_GEMFILE}.lock"
+  - "travis_retry gem update --system"
+  - "travis_retry gem install bundler -v 1.15.4"
+
+before_script:
+  # Set Sauce Labs username and access key. Obfuscated, purposefully not encrypted.
+  # Decodes to e.g. `export VARIABLE=VALUE`
+  - $(base64 --decode <<< "ZXhwb3J0IFNBVUNFX0FDQ0VTU19LRVk9YTAzNTM0M2YtZTkyMi00MGIzLWFhM2MtMDZiM2VhNjM1YzQ4")
+  - $(base64 --decode <<< "ZXhwb3J0IFNBVUNFX1VTRVJOQU1FPXJ1YnlvbnJhaWxz")
+
+script: 'ci/travis.rb'
+
+# This is not standard Travis; see `tasks/travis.rb`
+rvm_matrix:
+  - env: "GEM=railties"
+  - env: "GEM=ap,ac"
+  - env: "GEM=am,amo,as,av,aj,ast"
+    addons:
+      apt:
+        sources:
+          - sourceline: "ppa:mc3man/trusty-media"
+          - sourceline: "ppa:ubuntuhandbook1/apps"
+        packages:
+          - ffmpeg
+          - mupdf
+          - mupdf-tools
+    before_install+:
+      - "[[ -z $encrypted_8a915ebdd931_key && -z $encrypted_8a915ebdd931_iv ]] || openssl aes-256-cbc -K $encrypted_8a915ebdd931_key -iv $encrypted_8a915ebdd931_iv -in activestorage/test/service/configurations.yml.enc -out activestorage/test/service/configurations.yml -d"
+  - env: "GEM=as PRESERVE_TIMEZONES=1"
+  - env: "GEM=ar:mysql2"
+  - env: "GEM=ar:sqlite3"
+  - env: "GEM=ar:postgresql"
+  - env: "GEM=guides"
+  - env: "GEM=ac:integration"
+  - env: "GEM=aj:integration"
+    services+:
```

Comment:
I don't understand `+`. How does it affect `env: "GEM=aj:integration"`?

Reply:
https://github.com/rails/rails/blob/8c290251e8d98d16528a0158656e2777d7c60a00/tasks/travis.rb#L8-L10

i.e.

https://github.com/rails/rails/blob/8c290251e8d98d16528a0158656e2777d7c60a00/.travis.source.yml#L12-L14

\+

https://github.com/rails/rails/blob/8c290251e8d98d16528a0158656e2777d7c60a00/.travis.source.yml#L56-L57

=

https://github.com/rails/rails/blob/8c290251e8d98d16528a0158656e2777d7c60a00/.travis.yml#L128-L131

Reply:
Ahhh, nice 👍